### PR TITLE
Update Ruby YAML line references

### PR DESCRIPTION
**Pull request by benhayehudi**
## Description

Update `code` and `client` line references in the Ruby YAML files, related to https://github.com/Nexmo/nexmo-ruby-code-snippets/pull/54.

Don't merge until changes from the snippets have been merged.

## Deploy Notes

Notes regarding deployment the contained body of work. These should note any db migrations, etc.

### CODE REVIEW

File: _examples/application/create-application/ruby.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 code:
     source: .repos/nexmo/nexmo-ruby-code-snippets/application/create-application.rb
-    from_line: 17
-    to_line: 38
+    from_line: 14
+    to_line: 31
```
Comment (marklewin) [inline _examples/application/create-application/ruby.yml]: Shouldn't this be 14-35?

File: _examples/application/update-application/ruby.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
 code:
     source: .repos/nexmo/nexmo-ruby-code-snippets/application/update-application.rb
-    from_line: 19
-    to_line: 54
+    from_line: 16
+    to_line: 33
```
Comment (marklewin) [inline _examples/application/update-application/ruby.yml]: Should this be 16-51?

File: _examples/application/create-application/ruby.yml (hunk 1 above)
Comment (benhayehudi) [reply _examples/application/create-application/ruby.yml]: 👍

File: _examples/application/update-application/ruby.yml (hunk 2 above)
Comment (benhayehudi) [reply _examples/application/update-application/ruby.yml]: 👍

**Pull request merged**